### ISSUE TITLE
[Feature] <HID-input> extra controller actions

### ISSUE THREAD

**Issue by nieknooijens** [guusw/unnamed-sdvx-clone#22]
Hi there

It would be nice to add some extra controller actions:
- left dial will change difficulty
- right dial will scroll through songs
- start on the controller will start the song
- start + left-dial changes the high-speed value.

I'm currently trying to learn how the code works but since you're throwing functions around all over the place using delegates and function pointers it's going to take some more time for me to get used to this kind of code-style.

**Comment by GhettoGirl** [comment]
> start + left-dial changes the high-speed value.

did you meant BT-4 (hold) + red knob? because in KSM 1.50 you change the HiSPEED that way. did masaka changed that in 1.6x?

**Comment by nieknooijens** [comment]
@GhettoGirl start + left dial is the way it's changed in the original SDVX arcade.
(I only played gravity wars, don't know about infinite infection, will probably be the same)

**Comment by Drewol** [comment]
start + any knob should change the hi-speed in arcade sdvx for ksm i think it's start + red for hi-speed value and start + blue for hi-speed mode

**Comment by GhettoGirl** [comment]
@Drewol for KSM its BT-4 + red knob, not start

**Comment by guusw** [comment]
Yeah. My idea was to add buttons for these actions in the Input class so that these things can be controlled either by keyboard or controller (at least as far as this is possible).
